personal_assistant/pa_tag/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import main, add_tag, delete
+
+app_name = 'pa_tags'
+
+urlpatterns = [
+    path('', main, name='main'),
+    path('add_tag', add_tag, name='add_tag'),
+    path('<int:tag_id>delete', delete, name='delete'),
```

Comment:
Щоб все було, більш-менш, в одному стилі:
```suggestion
    path('add/', add_tag, name='create'),
    path('<int:tag_id>/', delete, name='delete'),
```